### PR TITLE
[JENKINS-59486] - Load icons in Plugin Manager while Jenkins is restarting

### PR DESCRIPTION
**Pull request by jsoref**
See [JENKINS-59486](https://issues.jenkins-ci.org/browse/JENKINS-59486).

### Proposed changelog entries

* Update Plugin manager pages to show icons while Jenkins is restarting

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There are at least 2 approvals for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [x] Proper changelog labels are set so that the changelog can be generated automatically
- [x] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387)) (not applicable)
- [x] If it would make sense to backport the change to LTS, a JIRA issue should exist and be labeled as `lts-candidate` (not applicable)

### CODE REVIEW

**Review by res0nance — APPROVED** [review]
Looks good could you add a screenshot to show exactly how the UI changes?

**Comment by jsoref** [comment]
I can try tomorrow. Basically before you often see broken images. With this, you don't.

**Review by varyvol — APPROVED** [review]

**Comment by varyvol** [comment]
Having the screenshot would be great, but the change looks good.

**Comment by MRamonLeon** [comment]
Isn't this image already loaded? I guess if the image is in the browser cache we don't need to do so. :man_shrugging:

**Comment by jsoref** [comment]
My flow is updating a plug-in or three followed by restarting Jenkins. This would be the first time some of those states are hit, and just after the state is hit, Jenkins restarts making the images unavailable.

**Comment by jsoref** [comment]
## Before 
Here's one example:
image

#### Steps
1. use the commit right before this commit
1. Open Incognito
1. visit http://localhost:9000/updateCenter/
1. Log in (using the default admin account)
1. click the checkbox to trigger a restart

The key portion is here:
image

> Restarting Jenkins ༕ Pending

Note: normally you have enough CSS that the top area doesn't corrupt, but I regularly don't have the images from the steps.

## After
image

**Review by MarkEWaite — APPROVED** [review]

**Comment by MarkEWaite** [comment]
I've marked this "ready  for merge" and will merge it within the next few days unless there are objections. 

@oleg-nenashev I assume this is a small enough change that it should not risk the delivery of the upcoming weekly release.  Is that assumption valid or should this wait to merge until early next week?

**Review by oleg-nenashev — APPROVED** [review]
The fix look good to me. I am not 100% comfortable about merging anything remotely risky into the next release taking 2 recent regressions, but I will leave this decision to other maintainers

**Comment by MarkEWaite** [comment]
I'll plan to merge after the next release.

**Review by MRamonLeon — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Merging it taking the merge notice from @MarkEWaite . Thanks @jsoref !

**Pull request merged**
